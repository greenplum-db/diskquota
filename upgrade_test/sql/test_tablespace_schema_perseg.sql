../../sql/test_tablespace_schema_perseg.sql