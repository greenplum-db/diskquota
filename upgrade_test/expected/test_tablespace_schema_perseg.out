../../expected/test_tablespace_schema_perseg.out